src/lib/constants.ts

```diff
@@ -0,0 +1,202 @@
+export const DUMMY_USERS = [
+  {
+    id: 1,
+    name: "Матвій Коломієць",
+    rating: 5.0,
+    reviews: 5,
+    profession: "Сантехнік",
+    district: "Київ",
+    avatar: "/static/boy.svg",
+    lastOnline: "2год тому",
```

Comment:
I am not sure that backend will give you `lastOnline` in form of Ukrainian string. I'd rather expect `lastOnline` to be `Date` and parse it client-side

Reply:
This is just dumb data that changes to normal when the backend is connected.

Reply:
Yes, and this is the problem, since you just make up some data, not knowing if backend can provide it to you. Now you will have to spend extra time refactoring your code once you start connecting to backend and see that you have mismatches.

Reply:
For future, I would try to avoid using dummy data in "production" code. Its better to build component the way it should work, and use dummy data in tools like storybook

Reply:
During development, I noticed that Figma uses only UI-friendly data and it turned out that long values ​​were breaking everything, so I added my own data to draw styles more correctly.

But I understood the idea that it is better to connect the backend first and work with valid data.